targets/unity-v2/source/ExampleTestProject/Assets/PlayFabEditorExtensions/Editor/Scripts/PlayFabEditorSDK/PlayFabEditorHttp.cs

```diff
@@ -68,6 +68,17 @@ private static void WriteResultFile(string url, Action<string> resultCallback, b
                 headers.Add("X-SecretKey", PlayFabEditorDataService.ActiveTitle.SecretKey);
             }
 
+            if(api.Contains("AAD"))
```

Comment:
Is there a way to be more explicit about this? I'm sure this works for now, but the string match feels bad.